blocks/medium-promo-block/package.json

```diff
@@ -5,9 +5,10 @@
   "author": "Brent Miller <brent.miller@washpost.com>",
```

Comment:
```suggestion
  "author": "Arc XP - Themes",
```

Reply:
Description should also be updated to be "Medium Promo Block"

Reply:
Got it. Thanks!